tutorials/users/ide/kernels-and-dependency-management.md

```diff
@@ -1 +1,108 @@
-# kernels-and-dependency-management
+# Kernels and Dependency Management
+
+## What is a Kernel?
+
+Kernels in Jupyter are self-contained, program language specific processes that run independently.  This makes them 
+useful for running Jupyter Notebooks and consoles with a specific programming language. 
+A kernel doesn't need to be configured to serve every purpose imaginable. Several kernels can be made, with none of the dependencies for one kernel affecting 
+the dependencies for another.
+
+The flexibility of having kernels configured and packaged for specific purposes is passed on to the user, allowing you a lot of choice in the notebook 
+environments available to you.  
+
+Below are examples of kernels available on Marble.  These will be updated or changed to fit the needs of the users.
+
+![Kernels](images/kernels-dependency/kernel-buttons.png)
+
+## How to install packages
```

Comment:
This section is about more than that though...

What about: "Customizing kernel packages", "Managing kernels", "Managing installed packages", etc.

Reply:
Changed title to "Managing packages in a kernel"